_includes/connect-zbt-1/getting-started/zbt-1-2-migrate-deconz-to-zha.md

```diff
@@ -0,0 +1,95 @@
+{% steps %}
+{% step "Stopping the deCONZ add-on" %}
+{% image "/static/img/connect-zbt-1/conbee-migrate-zha-02.png" "Stop the deCONZ add-on" %}
+{% stepContent %}
+
+1. In the [deCONZ add-on](https://my.home-assistant.io/redirect/supervisor_addon/?addon=core_deconz), make sure the **Start on boot** option is disabled.
+2. Select **Stop**.
+
+{% endstepContent %}
+{% endstep %}
+
+{% step "Removing the deCONZ integration" %}
+{% image "/static/img/connect-zbt-1/conbee-migrate-delete-deconz-integration.png" "Remove the deCONZ integration" %}
+{% stepContent %}
+
+1. Under [**Settings** > **Devices & services**](https://my.home-assistant.io/redirect/integrations/), select the deCONZ integration.
+2. Select the three dots menu and select **Delete**.
+
+{% endstepContent %}
+{% endstep %}
+
+{% step "Adding a new ZHA integration" %}
+{% image "/static/img/connect-zbt-1/conbee-migrate-zha-23.png" "Add a new ZHA integration" %}
+{% stepContent %}
+
+1. In the top-right corner, make sure the checkboxes **Show ignored integrations** and **Show disabled integrations** are selected.
+2. Check if the ZHA integration shows up.
+   - If the ZHA integration does not show up, select **Add integration**, then select **ZHA** from the list.
+
+{% endstepContent %}
+{% endstep %}
+
+{% step "Add the Zigbee Home Automation integration" %}
```

Comment:
nit: This title is a bit weird, since the previous one was already "Adding the ZHA integration".
It's fine to keep, but maybe there's something better?